runtime/kusama/src/lib.rs

```diff
@@ -1037,6 +1074,9 @@ construct_runtime! {
 
 		// Tips module.
 		Tips: pallet_tips::{Module, Call, Storage, Event<T>} = 36,
+
+		// Election pallet. Only works with staking, but placed here to maintain indices.
```

Comment:
not sure the comment make sense, this is just a normal pallet. nothing special here

Reply:
I'm trying to reason about where it should be located, because that will influence the order of `on_initialize` and such

Reply:
actually the indices is given by `= 36`, so we could put if after staking if wanted.
But maybe it is better to place if after Session actually, because it will use session information indirectly.

Also on_initialize order is reversed from declaration: https://github.com/paritytech/substrate/issues/6280

Reply:
hmm actually 9 is empty, can I put it there and there won't be a need to update the transaction version still?

Reply:
For now, I'll keep it as-is. As you said, not a big deal if we're 1 block off.

Reply:
> hmm actually 9 is empty, can I put it there and there won't be a need to update the transaction version still?

yes you can, but there is no real need to reuse index as well `= 36` is good.